### PR TITLE
install docker on build jenkins workers

### PR DESCRIPTION
**Pull request by estute**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR if it is code shared across multiple services and you don't own all of the services.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] If you are making a complicated change, have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?  Adding a new variable does not require the full list (although testing on a sandbox is a great idea to ensure it links with your downstream code changes).
  - [ ] Think about how this change will affect Open edX operators.  Have you updated the wiki page for the next Open edX release?
---

This will install Docker enginer/tools on build Jenkins workers.

### CODE REVIEW

**Comment by estute** [comment]
packer build for this branch: https://build.testeng.edx.org/job/build-packer-ami/13641/

File: playbooks/roles/jenkins_worker/defaults/main.yml [hunk 1]
```diff
@@ -1,6 +1,9 @@
 ---
 jenkins_user: "jenkins"
-jenkins_group: "jenkins"
+# the main group for the jenkins user
+jenkins_group: 'jenkins'
+# the groups that the jenkins user must belong to
+jenkins_groups: 'jenkins,docker'
```
Comment (jmbowman) [inline playbooks/roles/jenkins_worker/defaults/main.yml]: nit: stick with double quotes in both these values for consistency with the rest of the file?

**Comment by estute** [comment]
@jmbowman nit fixed. Aside from that, is this good to go for you?

**Review by jmbowman — APPROVED** [review]
Yup.

**Pull request merged**